ArduPlane/quadplane.cpp

```diff
@@ -2895,6 +2921,39 @@ void QuadPlane::vtol_position_controller(void)
     }
 }
 
+void QuadPlane::assign_tilt_to_fwd_thr(void) {
+#if QAUTOTUNE_ENABLED
+    if (plane.control_mode == &plane.mode_qautotune) {
+        // limiting forward pitch and using forward throttle or rotor tilt is incompatible with auto-tune
+        q_fwd_throttle = 0.0f;
+        q_fwd_nav_pitch_lim_cd = -aparm.angle_max;
+        return;
+    }
+#endif
+    if (is_positive(q_fwd_thr_gain) && is_positive(q_fwd_tilt_lim)) {
```

Comment:
0 would be a reasonable value of `q_fwd_tilt_lim` maybe `!is_negative(q_fwd_tilt_lim)`? However I think we should allow positive numbers too. I can see a case where a script sits and adjusts the parameter directly depending on the desired VTOL climb-rate so the wing helps the motors rather than just not doing the wrong thing.

Reply:
Interesting idea. I'll update it to  allow non positive Q_FWD_PIT_LIM values to be used.